scipy/signal/__init__.py

```diff
@@ -307,6 +307,7 @@
 from ._savitzky_golay import savgol_coeffs, savgol_filter
 from .spectral import *
 from .wavelets import *
+from .wigner import *
```

Comment:
Just importing the single function by name would be preferred for new code.

Reply:
Done.